### PR TITLE
fix(typings): Use `EditMessageOptions` and `ReplyOptions`

### PR DESCRIPTION
**Pull request by Tolfx**

### CODE REVIEW

**Comment by Tolfx** [comment]
image
image

**Review by xHyroM — APPROVED** [review]

File: typings/index.d.ts [hunk 1]
```diff
@@ -177,6 +177,13 @@ export class InteractionReply {
   has: boolean;
   isEphemeral: boolean;
   send(content: APIMessageContentResolvable | (MessageOptions & { split?: false }) | MessageAdditions): Promise<this>;
+  send(content: APIMessageContentResolvable | (MessageOptions & { split?: false }) | MessageAdditions): Promise<this>;
```
Comment (MatrixSystemPVP) [inline typings/index.d.ts:180]: Line already exist above

**Review by MatrixSystemPVP — CHANGES_REQUESTED** [review]
I think all 'MessageOptions' should be changed to 'ReplyOptions'.
And remove the one duplicated line.

(in the last commit)

**Comment by Tolfx** [comment]
Replaced `MessageOptions` to `ReplyOptions`, which @MatrixSystemPVP suggested, kinda more logical if you think about it.

**Review by AngeloCore — APPROVED** [review]

**Pull request merged**